src/Client/ConfigWindow.java

```diff
@@ -363,7 +363,7 @@ public ConfigWindow() {
 
   public void showConfigWindow() {
     this.synchronizeGuiValues();
-    frame.setVisible(true);
+    frame.setVisible(!isShown());
```

Comment:
Would prefer this method unmodified so that its name describes its function. This class would want a "toggleConfigWindow()" function.

Reply:
Makes sense - I took my best guess at when to call this vs toggle, let me know if you think this is appropriate or not:

* Tray handler: show
* Keybind: toggle
* Icon click: toggle

As far as the keybind, though the popup will grab focus you could still alt tab back to the game window and re-invoke the keybind to close it.